src/Api.php

```diff
@@ -958,11 +961,17 @@ public function setWebhook(array $params)
      *
      * @return Update
      */
-    public function getWebhookUpdates()
+    public function getWebhookUpdates($emitUpdateWasReceivedEvent = true)
```

Comment:
I think we could have a global config option to enable / disable events. Because the events could be applied for things like message sending, photo sending, etc. And we can't really have params for each of them to turn on/off. It would make sense to just enable / disable from the main.